include/deal.II/lac/sparse_matrix_tools.h

```diff
@@ -215,21 +215,15 @@ namespace SparseMatrixTools
       IndexSet locally_owned_dofs(total_sum);
       locally_owned_dofs.add_range(prefix_sum, prefix_sum + local_size);
 
-      Utilities::MPI::internal::ComputeIndexOwner::ConsensusAlgorithmsPayload
-        process(locally_owned_dofs, locally_active_dofs, comm, dummy, true);
-
-      Utilities::MPI::ConsensusAlgorithms::Selector<
-        std::vector<
-          std::pair<types::global_dof_index, types::global_dof_index>>,
-        std::vector<unsigned int>>
-        consensus_algorithm;
-      consensus_algorithm.run(process, comm);
-
       using T1 = std::vector<
         std::pair<types::global_dof_index,
                   std::vector<std::pair<types::global_dof_index, Number>>>>;
 
-      auto requesters = process.get_requesters();
+      std::map<unsigned int, IndexSet> requesters;
+      std::tie(std::ignore, requesters) =
```

Comment:
> s there anything that can be saved if one doesn't actually need the owning_ranks? The example in AffineConstraints does not need them. Nor does the use in extract_remote_rows() in sparse_matrix_tools.h. (Not arguing againt the patch mentioned above, since there are places where we use both return values. Just saying that there are places where we don't.)

@bangerth One can ignore the first return argument in user code. Adding, a third variant of the code seems to be a redundant, in particular since the major cost lie in the computation of the first argument, which is computed anyway.

Reply:
OK.

Reply:
Out of curiosity, other languages use `_` for variables you don't care about. We could do something like
```suggestion
      const auto [_, requesters] = ...
```
here. The only difference is that in the assignment to `std::ignore`, the object is destroyed right away, whereas in the assignment to `_`, it lives until the end of the block. Do you have feelings one way or the other? (For reference, I do not.)

Reply:
> Out of curiosity, other languages use _ for variables you don't care about. 

The problem of `_` is that you can use it only once (since it is essentially a variable name), which sucks.

Reply:
Once in every scope :-) And starting in C++26, even this isn't true any more: https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2023/p2169r4.pdf